static/includes/InstanceNetworkProcedure.md

```diff
@@ -0,0 +1,17 @@
+Configure the **Network** section settings to define how the instance connects to the host and external networks.
```

Comment:
This snippet is using bullets when they should be numbered steps to follow the parallel construction of the procedure this is part of. You have a few ways to get around this by rewording the options portion of this (lines 2, 6 and 8) and folding them into a single bullet list under line 1.
List the UI option as the bullet point, then provide the information about why to select the option.

Then you can change the bullet points in line 13 and 14 to either numbered steps or indented paragraphs under the option it pertains to. I'm not going to write that out here. We can discuss this option if you want more clarification on how to do that. In any case, too many bullet points here are not help with what action goes with what in this step.